contrib/perf/post-pr-comment.py

```
@@ -251,21 +360,79 @@ def main():
+                new_body += "\n" + previous_section
+
+                # Add back any older previous runs (limit to 3 most recent)
+                for prev_run in previous_runs[:2]:  # Keep at most 2 additional old runs
+                    new_body += prev_run + "\n"
+
+                # Add footer back
+                new_body += footer
+
+                if update_comment(owner, repo, existing_comment['id'], new_body, token):
+                    success_count += 1
+                    print(f"  ✓ Updated successfully", file=sys.stderr)
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)